CHANGELOG/CHANGELOG-3.4.md

```diff
@@ -8,6 +8,7 @@ Previous change logs can be found at [CHANGELOG-3.3](https://github.com/etcd-io/
 
 ### Package `clientv3`
 - Add [client backoff and retry config options](https://github.com/etcd-io/etcd/pull/17369).
+- [Make CGO_ENABLED configurable](https://github.com/etcd-io/etcd/pull/17422).
```

Comment:
On second look we might want to move these entries to a separate heading for `Go` rather than `clientv3` ?

```markdown
### Go
- [Make CGO_ENABLED configurable](https://github.com/etcd-io/etcd/pull/17422).

```